salesforce/events.go

```diff
@@ -0,0 +1,341 @@
+package salesforce
+
+import (
+	"context"
+	"fmt"
+	"log/slog"
+
+	"github.com/amp-labs/connectors/common"
+	"github.com/spyzhov/ajson"
+)
+
+type SFAPIResponseBody struct {
+	Id       string        `json:"id"`
+	Success  bool          `json:"success"`
+	Errors   []interface{} `json:"errors"`
+	Infos    []interface{} `json:"infos"`
+	Warnings []interface{} `json:"warnings"`
+}
+
+type EventChannel struct {
+	Id       string                `json:"Id,omitempty"` // nolint:tagliatelle
```

Comment:
I think it would be error prone if we just globally disable it. Usually majority of APIs do make them lowercase, but just some APIs to Uppercase. I think it would be better if we specify `tagliatelle` on fields that specifically tell this one can be Uppercase. @laurenzlong  @eberle1080 Let me know if you think globally disabling still better for some other reason.

Reply:
You can disable it for this file by adding it at the top of the file.

Reply:
@RajatPawar I actually don't want to add this to the top of the file either because there are multiple models which have different field name casing conventions. I have defined 10+ structs and some have Uppercase Fieldname and some have camelcase.

Reply:
I found out that I can add one at the beginning of the struct which will disable for all fields